src/hyperion/device_setup_plans/read_hardware_for_setup.py

```diff
@@ -19,13 +20,15 @@ def read_hardware_for_ispyb_pre_collection(
     undulator: Undulator,
     synchrotron: Synchrotron,
     s4_slit_gaps: S4SlitGaps,
+    aperture_scatterguard: ApertureScatterguard,
 ):
     LOGGER.info("Reading status of beamline parameters for ispyb deposition.")
     yield from bps.create(
         name=ISPYB_HARDWARE_READ_PLAN
     )  # gives name to event *descriptor* document
     yield from bps.read(undulator.current_gap)
     yield from bps.read(synchrotron.machine_status.synchrotron_mode)
+    yield from bps.read(aperture_scatterguard.ap_name)
```

Comment:
Should: The signal in `dodal` is called `aperture_name`, which I actually think is a lot better as `ap` is confusing without context

Reply:
not sure at the moment. there's both a property and a signal, which might be an anti-pattern 

![image](https://github.com/DiamondLightSource/hyperion/assets/56644812/6ee779d3-3d4b-4153-b874-e6e1c01d2234)